recipe/meta.yaml

```
@@ -15,55 +15,26 @@ source:
     - glew_win.patch  # [win]
 
 build:
-  number: 0
-  # Missing CDT packages on s390x and ppc64le
-  skip: True  # [s390x or ppc64le]
+  number: 1
   run_exports:
     - {{ pin_subpackage('glew', max_pin='x.x') }}
   missing_dso_whitelist:
-    - '**/libc.so.*'
-    - '**/libGLEW.*'
+    - "**/libc.so.*"
 
 requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
     - cmake
-    - make                               # [unix]
-    - ninja                              # [win]
+    - make  # [unix]
+    - ninja  # [win]
     - pkg-config
-    - m2-patch                           # [win]
-    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
-    - {{ cdt('libx11-devel') }}          # [linux]
-    - {{ cdt('libxext-devel') }}         # [linux]
-    - {{ cdt('libxrender-devel') }}      # [linux]
-    - {{ cdt('mesa-libgl-devel') }}      # [linux]
-    - {{ cdt('mesa-libegl-devel') }}     # [linux]
-    - {{ cdt('mesa-dri-drivers') }}      # [linux]
-    - {{ cdt('libxau-devel') }}          # [linux]
-    - {{ cdt('alsa-lib-devel') }}        # [linux]
-    - {{ cdt('gtk2-devel') }}            # [linux]
-    - {{ cdt('gtkmm24-devel') }}         # [linux]
-    - {{ cdt('libdrm-devel') }}          # [linux]
-    - {{ cdt('libxcomposite-devel') }}   # [linux]
-    - {{ cdt('libxcursor-devel') }}      # [linux]
-    - {{ cdt('libxi-devel') }}           # [linux]
-    - {{ cdt('libxrandr-devel') }}       # [linux]
-    - {{ cdt('pciutils-devel') }}        # [linux]
-    - {{ cdt('libxscrnsaver-devel') }}   # [linux]
-    - {{ cdt('libxtst-devel') }}         # [linux]
-    - {{ cdt('libselinux-devel') }}      # [linux]
-    - {{ cdt('libxdamage') }}            # [linux]
-    - {{ cdt('libxdamage-devel') }}      # [linux]
-    - {{ cdt('libxfixes') }}             # [linux]
-    - {{ cdt('libxfixes-devel') }}       # [linux]
-    - {{ cdt('libxxf86vm') }}            # [linux]
-    - {{ cdt('libxrandr') }}             # [linux]
-    - {{ cdt('alsa-lib') }}              # [linux]
+    - m2-patch  # [win]
   host:
-    - libglu                         # [linux]
+    - libgl-devel                # [linux]
+    - xorg-libx11                # [linux]
```

```suggestion
    - libglu                     # [linux]
    - libgl-devel                # [linux]
    - xorg-libx11                # [linux]
    - xorg-libxmu                # [linux]
    - xorg-libxi                 # [linux]
```
Might need to do something with the `GLEW_NO_GLU` variable but it seems like their build has options to use libGLU. They also list it as a runtime requirement.

Their build requirements are listed [here](https://github.com/nigels-com/glew/blob/master/README.md#build-1).